apps/molgenis-components/src/components/utils.test.ts

```diff
@@ -0,0 +1,79 @@
+import { describe, assert, test } from "vitest";
+import constants from "./constants";
+import { deepClone, flattenObject, isNumericKey } from "./utils";
+
+const {
+  CODE_0,
+  CODE_9,
+  CODE_BACKSPACE,
+  CODE_MINUS,
+  CODE_DELETE,
+  MIN_LONG,
+  MAX_LONG,
+} = constants;
+
+describe("isNumericKey", () => {
+  test("code is CODE_0 (48)", () => {
+    const keyboardEvent = { which: CODE_0 } as KeyboardEvent;
+    assert.isTrue(isNumericKey(keyboardEvent));
+  });
+
+  test("code is between CODE_0 and CODE_9", () => {
+    const keyboardEvent = { which: 50 } as KeyboardEvent;
+    assert.isTrue(isNumericKey(keyboardEvent));
```

Comment:
Overall question:

How does 'assert.isTrue' and 'isNumericKey' assert the question stated in the parentheses of 'test()' ?

Reply:
Numeric codes are between 48 and 57, so 50 is between that. The function isNumericKey returns a boolean, so the testing it uses the vitest function assert.isTrue to check the returned value.

Reply:
Sure, but I was only pointing out that the description is a bit vague.
Like '"code is CODE_0 (48)"  you do not actually assert that is is 48 and not 47 or 49. You always assert that it 'is numerical' 
So the description does not match what you test, that was my whole point with a few of these tests.